exercises/concept/currency-exchange/.meta/exemplar.py

```diff
@@ -52,13 +52,14 @@ def exchangeable_value(budget, exchange_rate, spread, denomination):
 	:return: int - maximum value you can get
 	"""
 
-	exchange_fee = (exchange_rate / 100) * spread
-	actual_rate = exchange_rate + exchange_fee
-	exchangeable_amount = int((budget / actual_rate) / denomination)
-	return exchangeable_amount * denomination
+    exchange_fee = (exchange_rate / 100) * spread
+    actual_rate = exchange_rate + exchange_fee
+    exchangeable_amount = int((budget / actual_rate) / denomination)
+    return exchangeable_amount * denomination
```

Comment:
Two spaces rather than 4?

Reply:
No...because otherwise it wouldn't run at all.  I think the issue here is that the docstring indentation is wonky.  Lines 48-53 need to be _de-indented_.

Reply:
Thanks, my Python Eye is not that sharp, but it looked wonky to me anyway.

Reply:
Clearly, mine isn''t as good as it should be, or you wouldn't need to be double-checking me!  😄   Many thanks for the through go through.  ❤️